src/Sylius/Bundle/AddressingBundle/Tests/Matcher/ZoneMatcherTest/fixtures.yaml

```diff
@@ -0,0 +1,42 @@
+Sylius\Component\Addressing\Model\Country:
+    country_PL:
+        code: PL
+
+Sylius\Component\Addressing\Model\Province:
+    province_{MA, LD, MZ, DS}:
+        code: <current()>
+        name: <current()>
+        abbreviation: <current()>
+        country: "@country_PL"
+
+Sylius\Component\Addressing\Model\ZoneMember:
+    member_province_{MA, LD, MZ, DS}:
+        code: <current()>
+    member_{poland_visegrad_group, poland_nato}:
+        code: PL
+    member_visegrad_group:
+        code: "VISEGRAD_GROUP"
+
+Sylius\Component\Addressing\Model\Zone:
+    zone_poland:
+        code: PL
+        name: Poland
+        type: province
+        members: ["@member_province_MA", "@member_province_LD", "@member_province_MZ", "@member_province_DS"]
+    zone_visegrad_group:
+        code: VISEGRAD_GROUP
+        name: Visegrad Group
+        type: zone
+        members: ["@member_poland_visegrad_group"]
+    zone_eu:
+        code: EU
+        name: European Union
+        type: zone
+        members: ["@member_visegrad_group"]
+        scope: 'only_eu'
```

Comment:
I'd be for using preexisting scopes ie `tax`/`shipping`/`all`, though the first two live in core it'd be less confusing than now

Reply:
Tbh I don't see a need for that change. Let's wait for more voices.

Reply:
I do agree with Jan, scopes are for application level zones re-use, strings like `only_eu` or `nato` don't match intended use.

This tests/specs double as documentation and misuse will be confusing for new developers.

For example I've once added relation between Zone and PaymentMethods, so I've add new custom scope `payment`.

Reply:
I agree also that it will be clearer to use preexisting scopes, I do not see any benefit of using the custom scopes here

Reply:
Done